src/utils/githubEventHelpers.ts

```diff
@@ -118,7 +118,39 @@ export async function modifyProjectIssueField(
     projectFieldOption,
     fieldId,
   }
-  await sendQuery(addIssueToGlobalIssuesProjectMutation, data, octokit);
+  await sendQuery(modifyProjectIssueFieldMutation, data, octokit);
+}
+
+export async function modifyDueByDate(
```

Comment:
why does this need a new function instead of `modifyProjectIssueField`? Its because it's not a single select field option